.github/workflows/build-docker-images.yml

```diff
@@ -97,7 +97,7 @@ jobs:
   latest-torch-deepspeed-docker:
     name: "Latest PyTorch + DeepSpeed"
     runs-on:
-      group: aws-g4dn-2xlarge-cache
+      group: aws-general-8-plus
```

Comment:
we are not using T4 runners anymore. Since the CPU runners work now for building this image, let's use it.